README.md

```diff
@@ -4,326 +4,117 @@
-- **ir_baseline**: simple information retrieval baseline that scores candidate responses with [TFIDF-weighted](https://en.wikipedia.org/wiki/Tf%E2%80%93idf) matching
-- **repeat_label**: basic class for merely repeating all data sent to it (e.g. for piping to a file, debugging)
-- **remote_agent**: basic class for any agent connecting over ZMQ
-- **local_human**: takes input from the keyboard as the act() function of the agent, so a human can act in the environment
-
-See the [directory](https://github.com/facebookresearch/ParlAI/tree/master/parlai/agents) for the complete list.
-
-
-
-### Examples
-
-[This directory](https://github.com/facebookresearch/ParlAI/tree/master/examples) contains a few particular examples of basic loops.
-
-- base_train.py: _very simple example shows the outline of a training/validation loop using the default Agent parent class_
-- display_data.py: _uses agent.repeat_label to display data from a particular task provided on the command-line_
-- display_model.py: _shows the predictions of a provided model on a particular task provided on the command-line_
-- eval_model.py: _uses the named agent to compute evaluation metrics data for a particular task provided on the command-line_
-- build_dict.py: _build a dictionary from a particular task provided on the command-line using core.dict.DictionaryAgent_
-
-### Tasks
+- [**core**](./parlai/core): contains the primary code for the framework
+- [**agents**](./parlai/agents): contains agents which can interact with the different tasks (e.g. machine learning models)
+- [**examples**](./parlai/examples): contains a few basic examples of different loops (building dictionary, train/eval, displaying data)
+- [**tasks**](./parlai/tasks): contains code for the different tasks available from within ParlAI
+- [**mturk**](./parlai/mturk): contains code for setting up Mechanical Turk, as well as sample MTurk tasks
+- [**messenger**](./parlai/messenger): contains code for interfacing with Facebook Messenger
+- [**zoo**](./parlai/zoo): contains code to directly download and use pretrained models from our model zoo
 
-Our first release included the following datasets (shown in the left panel), and accessing one of them is as simple as specifying the name of the task as a command line option, as shown in the dataset display utility (right panel):
-<p align=center><img width="100%" src="docs/source/\_static/img/tasks2.png" /></p>
 
-Over 20 tasks were supported in the first release, including popular datasets such as
-SQuAD, bAbI tasks, MCTest, WikiQA, WebQuestions, SimpleQuestions, WikiMovies, QACNN, QADailyMail, CBT, BookTest, bAbI Dialog tasks,
-Ubuntu, OpenSubtitles, Cornell Movie, VQA-COCO2014.
-Since then, several datasets have been added such as  VQAv2, VisDial, MNIST_QA, Personalized Dialog, InsuranceQA, MS MARCO, TriviaQA, and CLEVR. See [here](https://github.com/facebookresearch/ParlAI/blob/master/parlai/tasks/task_list.py) for the current complete task list.
+## Detailed Goals
```

Comment:
Yeah, having 3 or 4 separate bulleted lists under this is really awkward. I'm not sure we should have this section at all - but if we do, we should condense it better.

Reply:
I removed that section, added a few stuff in the first paragraph